kommitters/calendso
# Location visible before booking

## What does this PR do?



Fixes #2519

I edited the modal window where you create the event and when you select location **In person** you could add the city and the address of the meeting.

<img width="400" alt="image" src="https://user-images.githubusercontent.com/19562383/165296628-86286df2-accb-4c44-b623-7576712e8aa0.png">

I changed to the booking page when you have the option **In Person** it going to show you the city 
<img width="254" alt="image" src="https://user-images.githubusercontent.com/19562383/165299818-b104afdb-b02d-4092-b322-6e8ab67fa988.png">


## Type of change




## How should this be tested?



- Booking an event.
- Select a time for a meeting.
- You are going to see **In-person meeting** with the city info
